### ISSUE TITLE
Run axe against demo pages

### ISSUE THREAD

**Issue by scurker** [dequelabs/cauldron-react#115]
We should be dog fooding here and ensure that the components in our pattern libary don't raise any accessibility issues.

**Comment by schne324** [comment]
we could use https://github.com/lfre/lightkeeper (as @stephenmathieson brought up). The annoying part would be having to manually add each path (`"routes": ["/", "/components/foo", "/components/bar", "/components/my-new-component"]`) so we might want to come up with a solution for that bit before adopting it

**Comment by stephenmathieson** [comment]
I guess I closed #88 prematurely haha

Is it really that big of a deal to manually add routes to a config file? I don't think it'll change much, so it shouldn't be a burden. Also, even if we miss a page, having something testing for accessibility without deploying aget (or alike) seems like a win to me.

**Comment by stephenmathieson** [comment]
Mad science idea: write a webpack plugin that taps into our build and outputs lightkeeper config.

**Comment by schne324** [comment]
> Mad science idea: write a webpack plugin that taps into our build and outputs lightkeeper config.

☝️ you beat me to it! I was just responding to your previous comment with a similar idea -- have some build step generate the lightkeeper config!

sounds good to me

**Comment by stephenmathieson** [comment]
Looks outdated but promising: https://www.npmjs.com/package/react-router-to-array

**Comment by stephenmathieson** [comment]
Also if we do #177, we'll probably be maintaining the list of routes anyway
